### ISSUE TITLE
geom_title scales become distorted after applying ggplotly

### ISSUE THREAD

**Issue by chengvt** [ropensci/plotly#1050]
I still don't fully understand how geom_tile calculate the sizes of each tile. However, it seems that the same method is not transferred to ggplotly. 

```
dat <- data.frame(x = c(1,4,10), y = c(2, 5, 3), z = 1:3)
ggplot(dat, aes(x,y,fill=z)) + geom_tile() -> p
p
```
![ggplot](https://user-images.githubusercontent.com/7601097/27272024-12504d5a-5503-11e7-9ff6-fc3b10ffbe5b.png)

The scales become distorted after applying ggplotly.  
```
ggplotly(p)
```
![plotly](https://user-images.githubusercontent.com/7601097/27272032-19f8f980-5503-11e7-9a16-dbe7ed475bb2.png)

Is there a way to fix the size of each tiles after applying ggplotly?

**Comment by willdelcast** [comment]
@chengvt did you manage to get a workaround or is there a fix?

**Comment by chengvt** [comment]
@willdelcast No, I didn't look into it. The behaviours appear the same when there is no blank space between the tiles. 
```
# example
dat <- data.frame(x = c(1,2,3), y = c(1,2,3), z = 1:3)
ggplot(dat, aes(x, y, fill = z)) + geom_tile() -> p
p
ggplotly(p)
```
